src/main/java/at/hannibal2/skyhanni/data/HotmData.kt

```diff
@@ -509,14 +524,46 @@ enum class HotmData(
                 abilities.filter { it.isUnlocked }.forEach {
                     it.activeLevel = if (PEAK_OF_THE_MOUNTAIN.rawLevel >= 1) 2 else 1
                 }
+                val skymallItem = InventoryUtils.getItemsInOpenChest().find { it.stack.name == "§aSky Mall" || it.stack.name == "§cSky Mall" }?.stack ?: return@runNextTick
```

Comment:
move this check inside the parse function. And use the entry to identify the skymall perk

Reply:
A, I just do it myself